swift/test/Ice/location/AllTests.swift

```diff
@@ -6,6 +6,7 @@ import TestCommon
 
 func allTests(_ helper: TestHelper) async throws {
     // This function must be sendable as it can be concurrently executed by a TaskGroup.
+    nonisolated(unsafe) let helper = helper
```

Comment:
It sounds like TestHelper should be sendable.

Reply:
Yes